lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -555,7 +555,7 @@ def file_contents_on(host, file_path)
             if host['platform'] =~ /windows/
               file_path.gsub!('/', '\\')
```

Comment:
that substitution is not done in file_exists_on

Reply:
It looks to work without, should I still add it there?